hknweb/tours/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.conf import settings
+from .models import DepTour
+from django.contrib.auth.models import User
+from django.forms.widgets import SelectDateWidget
+# from django.contrib.admin.widgets import AdminDateWidget
+
+import datetime
+
+
+class TourRequest(forms.ModelForm):
+    date = forms.DateField(widget=SelectDateWidget(), label='Desired Date')
```

Comment:
It's currently possible to request a dept tour before the current time. We should add some sort of check in so that there's less confusion on both sides of the dept tour request e.g. Rehan reads the time is for last month and not this month and he has to reach out to the dept tour requester to confirm that they meant this upcoming month. 

This applies to both the mm/dd/yyyy:
![hknconnie dept tour pr date submitted time before current time](https://user-images.githubusercontent.com/46059916/76896508-2e7efa00-684f-11ea-8d6b-1f9425ff6049.PNG)
and to the hh/mm on the next line down:   
![hknconnie dept tour pr date submitted time before current time today](https://user-images.githubusercontent.com/46059916/76896535-3b9be900-684f-11ea-97c2-81bdfcf8ba53.PNG)